client/packages/purchasing/src/common/Inputs.tsx

```diff
@@ -67,20 +61,21 @@ export const useInputComponents = (
       numericInput: (
         label: LocaleKey,
         value: number | null | undefined,
-        options: NumericInputProps = {}
+        options: NumericTextInputProps = {}
       ) => {
         const { onChange, max, decimalLimit, endAdornment } = options;
         return (
           <NumInputRow
             key={label}
             disabled={disabled}
-            label={t(label)}
             value={value ?? 0}
             onChange={onChange}
             max={max}
             decimalLimit={decimalLimit}
             endAdornment={endAdornment}
             isVerticalScreen={isVerticalScreen}
+            {...options}
+            label={t(label)}
```

Comment:
Label has to go *after* the options, as the type of the "label" prop *in* options rejects the label value otherwise.